tooling/generate_instances_types.py

```diff
@@ -0,0 +1,18 @@
+import boto3
+
+session = boto3.Session(region_name="us-east-1", profile_name="sch-gov")
+pricing = session.client('pricing')
+
+
+response = pricing.get_attribute_values(ServiceCode='AmazonEC2', AttributeName='instanceType')
+
+
+path = 'type.py'
```

Comment:
When will we generate this? Build time?

Reply:
I didn't think too much on it, now I generate it manually but it's a good idea automate this...

Reply:
@joaoqalves I will do this in another PR

Reply:
Ok. I'm not sure if this is a good idea as it is. I'd rather use meta-programming to create classes on the fly or Jinja2 templating

Reply:
@joaoqalves as you say sure it's better jinja2. I will remove this of this PR and create another one to do it better. I think are two different things. Sorry for mix it.